app/components/LegoReactions/index.tsx

```diff
@@ -4,10 +4,13 @@ import type { ID } from 'app/store/models';
 import type Emoji from 'app/store/models/Emoji';
 import type { ReactionsGrouped } from 'app/store/models/Reaction';
 import type { ContentTarget } from 'app/store/utils/contentTarget';
+import { CurrentUser } from 'app/store/models/User';
```

Comment:
Please import with the type annotation
```suggestion
import type { CurrentUser } from 'app/store/models/User';
```